crmprtd/moti/normalize.py

```diff
@@ -26,8 +26,11 @@ def normalize(iterable):
     file_stream = iterable_to_stream(iterable)
     et = xmlparse(file_stream)
     et = transform(et)
-
     obs_series = et.xpath("//observation-series")
+    if not len(obs_series[0]):
```

Comment:
I think that your test here doesn't account for the fact that there could be (and I *think* there often are) multiple `observation-series` elements in a data stream. So if the first series is empty, your test will skip all subsequent series, whether or not they are empty. I'll push a change that demonstrates this in the tests.